### PR TITLE
docs: v2.0.x: update community tax to 2%

### PR DESCRIPTION
**Pull request by youngjoon-lee**
Same as the PR #614, but based on the `release/v2.0.x` branch.

### CODE REVIEW

**Review by gyuguen — APPROVED** [review]
LGTM

File: .gitbook/resources/validator-faq.md [hunk 1]
```diff
@@ -215,11 +215,14 @@ This mechanism aims to incentivize non-empty block proposals, better networking
 Let's take a concrete example to illustrate the aforementioned concept.
 In this example, there are 10 validators with equal stake.
 Each of them applies a 10% commission rate and has 20% of self-delegated MEDs.
-Now comes a successful block that collects a total of `1005 MEDs` in fees.
+Now comes a successful block that collects a total of 1025.51020408 MED in fees.
 
-First, a community tax is applied, but it is `0%` on the Panacea for now.
+First, a 2% tax is applied. The corresponding MEDs go to the reserved community pool.
```
Comment (H4NLee) [inline .gitbook/resources/validator-faq.md:220]:
```suggestion
First, a 2% tax is applied. The corresponding MED go to the reserved community pool.
```

**Review by H4NLee — APPROVED** [review]
I think you are gonna change `MEDs` to `MED`, so leave a comment.
LGTM

**Review by audtlr24 — APPROVED** [review]
LGTM

**Pull request merged**